.python_package/setup.cfg

```diff
@@ -0,0 +1,44 @@
+[metadata]
+name = gridtools-cpp
+author = ETH Zurich
+author_email = gridtools@cscs.ch
+license_files = LICENSE
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+url = https://gridtools.github.io
+project_urls = 
+	Source Code = https://github.com/GridTools/gridtools
+platforms = any
+version = 2.2.1a2
+	
+classifiers = 
+	Development Status :: 5 - Production/Stable
+	License :: OSI Approved :: BSD License
+	Intended Audience :: Science/Research
+	Operating System :: OS Independent
+	Programming Language :: Python :: 3 :: Only
```

Comment:
Please, sort the classifiers alphabetically. Optionally, you might want to add the same topic classifiers used in gt4py:
```
    Topic :: Scientific/Engineering :: Atmospheric Science
    Topic :: Scientific/Engineering :: Mathematics
    Topic :: Scientific/Engineering :: Physics
```